src/components/Loans/OpenLoans/helpers/isOverridePossible/isOverridePossible.js

```diff
@@ -0,0 +1,19 @@
+import overridePossibleMessages from './overridePossibleMessages';
+
+export default (errorMessage) => {
+  return overridePossibleMessages.reduce((data, { message, showDueDatePicker }) => {
```

Comment:
can we use foeEach instead of reduce in this case?

Reply:
Done.